File: app/src/main/java/com/krystianwsul/checkme/domainmodel/LocalCustomTimeRelevance.java
package com.krystianwsul.checkme.domainmodel;

import android.support.annotation.NonNull;

import com.krystianwsul.checkme.domainmodel.local.LocalCustomTime;

public class LocalCustomTimeRelevance {
    @NonNull
    private final LocalCustomTime mLocalCustomTime;

    private boolean mRelevant = false;

    LocalCustomTimeRelevance(@NonNull LocalCustomTime localCustomTime) {
        mLocalCustomTime = localCustomTime;
    }

    void setRelevant() {
        mRelevant = true;
    }

    boolean getRelevant() {
        return mRelevant;
    }

    @NonNull
    LocalCustomTime getLocalCustomTime() {
        return mLocalCustomTime;
    }
}

File: app/src/main/java/com/krystianwsul/checkme/domainmodel/LocalCustomTimeRelevance.kt
package com.krystianwsul.checkme.domainmodel

import com.krystianwsul.checkme.domainmodel.local.LocalCustomTime

class LocalCustomTimeRelevance(val localCustomTime: LocalCustomTime) {

    var relevant = false
        private set

    fun setRelevant() {
        relevant = true
    }
}
